pyciemss/ouu/ouu.py

```diff
@@ -231,6 +228,7 @@ def update_progress(xk):
             niter=self.maxiter,
             minimizer_kwargs=minimizer_kwargs,
             take_step=take_step,
+            # callback=self.progress_hook,
```

Comment:
Could you please remove this commented out code?

Reply:
Yes